src/lib/kit/components/Inputs/TableArrayInput/TableArrayInput.tsx

```diff
@@ -138,6 +164,15 @@ export const TableArrayInput: ArrayInput = ({spec, name, arrayInput, input}) =>
         return [idxColumn, ...columns, removeColumn];
     }, [name, spec, onItemRemove, parentOnChange, input.value]);
 
+    const rowClassNames = React.useCallback(
+        (item: ObjectValue) => {
```

Comment:
let's fix the Table data prop
we need to change from string[] to {key: string}[]